packages/integration-tests/src/views/bareMetalDiscoveryIsoModal.js

```diff
@@ -108,7 +127,7 @@ export const bareMetalDiscoveryIsoModal = {
     }
   },
   waitForIsoGeneration: (timeout = Cypress.env('GENERATE_ISO_TIMEOUT')) => {
-    cy.get('h4', { timeout: timeout }).should('contain', 'Discovery ISO is ready to download');
+    cy.get('h4', { timeout: timeout }).should('contain', ' Discovery ISO is ready for download');
```

Comment:
It would be better to remove the initial space.

Reply:
Ok, so I'll do  a PR for that is the lib

Reply:
You can remove it from here and it would work even if the lib is not updated yet.